### PR TITLE
fix: error when trying to save 'other education'

### PR DESCRIPTION
**Pull request by dyudyunov**
When user selects "Other Education" and clicks save - error occurs.

It's caused by the wrong API call payload value `'o'`
Fix: use the `'other'` payload value instead (the valid one).

Related PRs:
- [Nutmeg](https://github.com/openedx/frontend-app-profile/pull/665)
- [Olive](https://github.com/openedx/frontend-app-profile/pull/666)
- [Palm](https://github.com/openedx/frontend-app-profile/pull/746)

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @dyudyunov!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by e0d** [comment]
@dyudyunov looks like there are some test failures.

**Comment by dyudyunov** [comment]
> @dyudyunov looks like there are some test failures.

yeah, I should definitely update snapshots, thanks

**Comment by dyudyunov** [comment]
@e0d I've pushed the updated snapshots, please, run tests

**Comment by mphilbrick211** [comment]
Hi @dyudyunov - is this PR ready for review?

**Comment by dyudyunov** [comment]
> Hi @dyudyunov - is this PR ready for review?

Hello @mphilbrick211!
Yes, it's ready

**Comment by mphilbrick211** [comment]
Flagging this for review/merge @hurtstotouchfire.

**Comment by mphilbrick211** [comment]
Friendly ping on this @hurtstotouchfire :)

**Comment by dyudyunov** [comment]
4 months - is the average time for my PRs to be reviewed, looks like this one will be merged soon... 🙃

**Comment by dyudyunov** [comment]
JFYI 

I'm going to create a backport PR for the Palm.
The Nutmeg PR will be probably closed because the release isn't supported anymore 😑

**Comment by dyudyunov** [comment]
So the Olive backport loses its chances to get the fix too 😑

Please review the master and at least the backport to palm.master 🙏

**Comment by mphilbrick211** [comment]
Hi @openedx/2u-aperture! Would someone mind taking a look at this? Thanks!

**Comment by dyudyunov** [comment]
friendly ping

**Comment by jsnwesson** [comment]
Hey all, so sorry for the delay on this. I'll be adding this PR to the top of the queue for next week. I'll also be going through any other open PRs in Profile this week to get things ready for our kanban/platform workstream that's starting next Monday. This PR is honestly easy enough that I'll argue for getting it approved this week.

**Review by jsnwesson — APPROVED** [review]

**Comment by openedx-webhooks** [comment]
@dyudyunov 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-profile&entry.867055334=https://github.com/openedx/frontend-app-profile/pull/667&entry.1484655318=https://github.com/dyudyunov&entry.752974735=2023-01-11+15:16&entry.1917517419=2023-07-03+17:22&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by jsnwesson** [comment]
@mphilbrick211 I just realized that I should've messaged you first before merging these. 😅 Once approved, do you need me to also merge them since I have write access?

**Comment by mphilbrick211** [comment]
@jsnwesson - yes, thank you so much!

**Pull request merged**
